supply-chain-security/compromises/README.md

```diff
@@ -0,0 +1,42 @@
+# Catalog of Supply Chain Compromises
+
+
+This repository contains links to articles of software supply chain
+compromises. The goal is not to catalog every known supply chain attack, but 
+rather to capture many examples of different kinds of attack, so that we 
+can better understand the patterns and develop best practices and tools.
+
+We welcome additions to this catalog by 
+[filing an issue](https://github.com/cncf/sig-security/issues/new/choose) or
+[github pull request](https://github.com/cncf/sig-security)
+
+
+| Name              | Year               | Type of compromise    | Link        |
+| ----------------- | ------------------ | ------------------    | ----------- |
+| [electron-native-notify] | 2019 | Source Code Compromise | [1](https://blog.npmjs.org/post/185397814280/plot-to-steal-cryptocurrency-foiled-by-the-npm)[2](https://komodoplatform.com/update-agama-vulnerability/)|
+| [ShadowHammer](compromises/2019/shadowhammer.md) | 2019 | Multiple steps | [1](https://www.csoonline.com/article/3384259/asus-users-fall-victim-to-supply-chain-attack-through-backdoored-update.html), [2](https://securelist.com/operation-shadowhammer/89992/) |
```

Comment:
broken links:  since README is in `compromises` directory, need to remove `compromises/` from links